src/frontend/src/modules/Client/DeferredPayments.vue

```diff
@@ -40,6 +41,9 @@
             </md-table-cell>
             <md-table-cell md-label="Rates" md-sort-by="rate_count">
               {{ item.rate_count }}
+              /* 
+              - #606 - Enhancing functionality by commenting out the following section 
+  and implementing row click event instead of the eye button click event.
               <div
```

Comment:
Please don't comment out code. You can just remove it. We have Git Blame/Git History if we ever need this code block back 😉